sipa/model/wu/user.py

```diff
@@ -324,6 +325,20 @@ def userdb_status(self):
     def userdb(self):
         return self._userdb
 
+    @active_prop
+    def finance_balance(self):
+        return sum(t.wert for t in self._nutzer.transactions)
```

Comment:
@fgrsnau that's what I mean. Isn't that wrong since `haben` and `soll` can be switchet, causing the effective sign to be wrong?